Hide / Reset Version Button (when clicked)

arrmo opened the pull request.

tagyoureit commented:
Basic functionality is all there.  I have one small-ish fix on my side.  Two picky notes.  

1. On the reset layout it should just be `false` (the boolean) as the value and not `"false"` (the string).
1. When the page first loads, you can see the orange box blip before it is hidden by the jquery script.  Can you include the tag to hide it in the HTML element itself?  Then it should still be made visible when it receives the socket call.

thanks!

arrmo commented:
Agreed - thanks! I changed the logic a bit, "reversed" it so the starting point is hidden. And corrected the string - had it right one place, wrong the other.

Update is in PR #43.
